### DISCUSSION TITLE
Win32

### DISCUSSION THREAD

**Discussion by zvezdochiot** [ImageProcessing-ElectronicPublications/mfbdjvu#3] [General]
## Hi all.

:confused: Needed build for `win32`. My `mingw` refuses to compile `djvulibre`, so you can't compile `mfbdjvu` without it either. Any builds are welcome.

**Comment by plzombie** [comment]
Add VS project files in commit https://github.com/ImageProcessing-ElectronicPublications/mfbdjvu/commit/3540e27edce5f1e2739c78992b5e011b5fdbe21e
Add binaries to [release 1.2](https://github.com/ImageProcessing-ElectronicPublications/mfbdjvu/releases/tag/1.2)
